project/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -135,6 +136,21 @@ private DirectoryCandidate extractCandidateInformation(String candidateId,
                                   isIncumbent);
   }
 
+  /**
+   * Takes an {@code Entity} object which represents an elections, and returns true if
+   * this election is deemed relevant to the given address. If the {@code listAll} parameter
+   * is true, then the election is always deemed relevant.
+   */
+  private boolean isRelevantElection(Entity election, String address, boolean listAll) {
+    if (listAll) {
```

Comment:
So this is just always returning true for now? If so, you can just do return true with the TODO right above it